hack/common.sh

```diff
@@ -159,9 +184,16 @@ os::build::build_binaries() {
     for platform in "${platforms[@]}"; do
       os::build::set_platform_envs "${platform}"
       echo "++ Building go targets for ${platform}:" "${targets[@]}"
-      go install "${goflags[@]:+${goflags[@]}}" \
-          -ldflags "${version_ldflags}" \
-          "${binaries[@]}"
+      for artifact in ${binaries[@]}; do
```

Comment:
making upstream test into a utility is doable, but likely will meet a little pushback.